src/routes/index.ts

```diff
@@ -26,6 +30,28 @@ export const loginRateLimiter = createRateLimitMiddleware(
 	{},
 );
 
+export const registerDeviceStateEvent = (pathToUuid: _.PropertyPath) => {
```

Comment:
Is this used by anyone else? Why export it?

Reply:
Not in `open-balena-api` but I was thinking of hooking the V1 state endpoint in `resin-api`

Reply:
Cool, if it's useful elsewhere, sounds good 👍